tendrl/gluster_integration/objects/volume/atoms/start_profiling/__init__.py

```diff
@@ -25,5 +27,12 @@ def run(self):
                     err
                 )
             )
+        while True:
+            volume = NS.gluster.objects.Volume(
+                vol_id=vol_id
+            ).load()
+            if volume.profiling_enabled == "True":
```

Comment:
can we start using  "yes" or "no" everywhere boolean?

Reply:
We can I feel. will need to check the dependency at least in this case and change in all references. Will update the PR soon.